extensions/fairifier-rdf-extension/dtls/fairifier/FileSystemRdfSkeletonImp.java

```diff
@@ -0,0 +1,92 @@
+package org.dtls.fairifier;
```

Comment:
our domain is 'nl.dtls'

Reply:
The fairifier now uses org, but i could change that